gcn/notices/fermi/gbm/strict/alert.schema.json

```diff
@@ -0,0 +1,174 @@
+{
+  "$id": "https://gcn.nasa.gov/schema/main/gcn/notices/fermi/gbm/strict/alert.schema.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "GCN Schema for Fermi GBM Alert Notices",
+  "description": "This schema is a stricter version of the more permissive main schema, https://gcn.nasa.gov/schema/main/gcn/notices/fermi/gbm/alert.schema.json.  The Kafka server uses the main schema for new notices.  Each Alert notice (classic numeric type 110) must validate against the main schema and should also validate against this one.  Examples in this directory come in two versions referencing the main and the strict schemas respectively.  The intent of the strict schema is to help understanding the Alert notice, since it better describes what content to be expected.",
+  "type": "object",
+  "properties": {
+    "$schema": {
+      "enum": [
+        "https://gcn.nasa.gov/schema/main/gcn/notices/fermi/gbm/alert.schema.json",
+        "https://gcn.nasa.gov/schema/main/gcn/notices/fermi/gbm/strict/alert.schema.json"
```

Comment:
This shouldn't reference a particular version of the schema, but should be a relative reference.

Reply:
The exact value here should actually be:

```json
"$schema": { "type": "string", "format": "uri" }
```
The only purpose for defining the `$schema` attribute in the properties section is so that, when used with `'unevaluatedProperties': false`, the validator knows that there should be a `$schema` in a notice